README.md

```diff
@@ -70,12 +70,11 @@ The Apache Spark Connector for SQL Server and Azure SQL is based on the Spark Da
 
 To include the connector in your projects download this repository and build the jar using SBT.
 
-### Write to a new SQL Table
-#### Important: using the `overwrite` mode will first DROP the table if it already exists in the database. Please use this option with due care to avoid unexpected data loss!
+### Write to SQL Table
 ```python
 server_name = "jdbc:sqlserver://{SERVER_ADDR}"
 database_name = "database_name"
-url = server_name + ";" + "databaseName=" + database_name + ";"
+url = servername + ";" + "database_name=" + dbname + ";"
```

Comment:
Might be mistaken but I think databaseName is a needed to properly build the JDBC string
https://docs.microsoft.com/en-us/sql/connect/jdbc/building-the-connection-url?view=sql-server-ver15

@mokabiru can you double check this line change?

Reply:
@rajmera3 Apologies my file sync on OneDrive messed it up. My last commit (commit a7cf3195274f2a790c000485b74d3ae5141c22e7) has the updated README from previous commits including Arvind's.

Reply:
Commit Link: https://github.com/microsoft/sql-spark-connector/pull/12/commits/a7cf3195274f2a790c000485b74d3ae5141c22e7